config/menus/maps.cfg

```diff
@@ -776,13 +776,57 @@ newgui maps [ ui_stay_open [ octapaks [
     ui_strut 0.5
     ui_font super [
         ui_center [
+            show_force_button = 0
+            if (&& (getclientpriv $getclientnum $vetolock) (=s (getmastermode 1) "veto")) [
+                show_force_button = 1
+            ]
```

Comment:
If you are showing the force button, then you probably shouldn't have the normal vote button which also forces. Perhaps have a button to force a new vote on the selection screen and a button to force your current vote on the votes screen, or a similar arrangement.

Reply:
but... the normal Vote button doesn't force?

Reply:
Yes, it does, if mastermode is set to veto and you have veto privs.

Reply:
oh... well that smth we could change, Imo you should still be able to normally vote, right?

Reply:
No, why should you? `veto` mode is something rarely used anyway.

Reply:
the question is: why should we NOT do it